src/partials/notice-banner.hbs

```diff
@@ -9,6 +9,7 @@
       page.componentVersion.asciidoc.attributes.olderVersion))
     }}
     <div class="notice-banner">
+      <button class="notice-banner-close-button" title="Close the notice banner">&times;</button>
       {{#if page.componentVersion.asciidoc.attributes.endOfLife}}
       <p>This version of the product has reached 
```

Comment:
I know you didn't change this, but since it's close to the new work, maybe you could move this into a labels file, too.